### ISSUE TITLE
Please someone take out that dramatic zoom-in effect on clicking the choropleths

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#1141]
It makes it hard to quickly interact with the map and compare regions to regions. Should be super fast.

@misaugstad will you save us?

**Comment by misaugstad** [comment]
@jonfroehlich 
Yes I can handle this quickly. So there are two things that happen... We center on that neighborhood and we zoom really close to it. You said to remove the zoom, but I just want to check: do you want it totally removed, or just reduced? And do you want to keep or remove the centering on that neighborhood?

**Comment by misaugstad** [comment]
I think that we should remove the center as well, so we don't disorient users at all. I don't see a good reason to center. So here is the update...

Before clicking on a region (looks the same):
![pre-click](https://user-images.githubusercontent.com/6518824/31567742-eef2f78a-b03e-11e7-9480-f5d5ff0467e2.png)

How it used to look after clicking:
![yes-zoom](https://user-images.githubusercontent.com/6518824/31567756-fd9cc086-b03e-11e7-9133-b6c239923673.png)

How it looks with the changes I made:
![no-zoom](https://user-images.githubusercontent.com/6518824/31567761-02e18220-b03f-11e7-8fca-ce28a4622cda.png)

And I updated this for every choropleth that we have. If this looks like the right behavior @jonfroehlich @manaswis then I will submit a PR.
